src/main/java/sample/main2.java

```diff
@@ -1,18 +1,17 @@
 package sample;
 
+import sample.Utils.ShoppingCart;
 import sample.Utils.ProductDao;
 import sample.Utils.UserDao;
-import sample.Utils.UserVerifier;
 import sample.entity.Product;
 import sample.entity.User;
-import sample.entity.UserType;
 
 public class main2 {
     public static void main(String[] args) {
         User user = new User();
-        user.setUsername("Marcelino");
-        user.setPassword("password");
-        user.setUserType(UserType.CUSTOMER);
+        user.setUsername("Norocosul");
+        user.setPassword("000000");
+//        user.setUserType(UserType.CUSTOMER);
```

Comment:
Usually, commented code is not good. You have versioning now. If you need something that you deleted you can always recover it

Reply:
This class is just for test and debug.